.gitignore

```diff
@@ -55,3 +55,4 @@ massif-*
 /GTAGS
 /TAGS
 /tags
+/cscope*
```

Comment:
Please keep the list in alphabetical order (`cscope` before `tags`), that makes it easier to maintain.

Ignore lists should not be too broad. I think this should be `/cscope*.out`, and maybe `/cscope.files` although I don't see the point of that file (see how `Makefile` generates `TAGS`).

Reply:
Thanks @gilles-peskine-arm I will update the PR shortly.